R/pkg/R/generics.R

```diff
@@ -942,6 +942,14 @@ setGeneric("countDistinct", function(x, ...) { standardGeneric("countDistinct")
 #' @export
 setGeneric("crc32", function(x) { standardGeneric("crc32") })
 
+#' @rdname create_array
```

Comment:
this is also `###################### Expression Function Methods ##########################` might not be the right place